templates/todo/projects/nodejs-mongo-aks/src/api/manifests/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todo-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: todo-api
+  template:
+    metadata:
+      labels:
+        app: todo-api
+    spec:
+      containers:
+        - name: todo-api
+          image: crppnnadqdq7owqmxjqvwcid425o.azurecr.io/todo-nodejs-mongo-aks/api:latest
```

Comment:
We'll have to turn this into a template and make this a parameter we can replace.

Reply:
Yes, this is in the works, and still just a draft at this point.  Updates coming soon.